ORMSolution/ORMNUnit/NUnitSetupFixture.cs

```diff
@@ -13,28 +14,31 @@ public void Initialize()
         {
             var memoryEntityTables = new List<string>()
             {
-                "MemoryEntityTables/USERS.xml",
-                "MemoryEntityTables/ORGANISATIONS.xml"
+                $"{nameof(ORMNUnit)}.MemoryEntityTables.USERS.xml",
```

Comment:
nameof(ORMNUnit) misschien in een variable zetten? Past achteraf toch makkelijker aan :)

Reply:
Hm, ja idee was nameof(namespace) te doen omdat dat toch het naam van het project is, dus dan is het al altijd goed. Maar via een variable is ook mogelijk. Hebben het morgen hier wel even over :)